common/configurable/pom.xml

```diff
@@ -58,13 +63,18 @@
         </dependency>
         <dependency>
             <groupId>org.hamcrest</groupId>
-            <artifactId>hamcrest-core</artifactId>
+            <artifactId>hamcrest-all</artifactId>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
```

Comment:
Do we really need a new dependency for a single test? Do we have approval for this?

Reply:
Yes we have approval. I had the same thought and tried replacing it with a simple mechanism, but I could not get the growth test to pass. I did spend some time at it, but ultimately decided it wasn't worth any more effort. 

This library has some [really nice features](https://github.com/awaitility/awaitility/wiki/Usage) which I fully expect will be useful elsewhere.

Reply:
OK